docs/package_integration/uds-package-practices.md

```diff
@@ -120,12 +112,15 @@ In practice, this results in the following for the second release of a package f
 17.2.1-uds.1
 ```
 
-## General
-
-And in addition to the above, packages generally:
+## Documenting Exceptions
 
-- **Must** be capable of operating within an internet-disconnected (air-gapped) environment.
+If you are unable to meet a requirement or best practice, document the justification in your package repository `docs/package_exceptions.md`
 
-- **Must** not make the assumption that the `expose` interfaces are accessible to the bastion or pipeline deploying the package (i.e. `*.uds.dev`).  If web requests need to be made they should be done through a `Job` or `./uds zarf tools kubectl exec` as appropriate.
+Some examples of valid justifications would be:
+- ✅ The last step of deployment requires a click-ops interaction and cannot be automated through IaC
+- ✅ Upstream application does not expose metrics endpoint to integrate
 
-- **Must** be maintained by a resourced team that is explicitly defined as maintaining the project (i.e. in `CODEOWNERS`).
```

Comment:
I understand that this is sort of replaced above, but I dont like the implications of (at least for any internal to the company developed package) "resourced team" no longer being in the language.

Reply:
I added a small section to articulate!